cmd/create/breakglasscredential/cmd.go

```diff
@@ -0,0 +1,82 @@
+package breakglasscredential
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/breakglasscredential"
+	"github.com/openshift/rosa/pkg/externalauthprovider"
+	"github.com/openshift/rosa/pkg/interactive"
+	"github.com/openshift/rosa/pkg/ocm"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+var breakGlassCredentialArgs *breakglasscredential.BreakGlassCredentialArgs
+
+const argsPrefix string = ""
+
+var Cmd = &cobra.Command{
+	Use:     "break-glass-credential",
+	Aliases: []string{"break-glass-credentials", "breakglasscredential", "breakglasscredentials"},
+	Short:   "Create a break glass credential for a cluster.",
+	Long:    "Create a break glass credential for a hosted control plane cluster with external authentication enabled.",
+	Example: `  # Interactively create a break glass credential to a cluster named "mycluster"
+  rosa create break-glass-credential --cluster=mycluster --interactive`,
+	Run:    run,
+	Hidden: true,
+	Args:   cobra.NoArgs,
+}
+
+func init() {
+	flags := Cmd.Flags()
+
+	ocm.AddClusterFlag(Cmd)
+
+	interactive.AddFlag(flags)
+	breakGlassCredentialArgs = breakglasscredential.AddBreakGlassCredentialFlags(Cmd, argsPrefix)
+}
+
+func run(cmd *cobra.Command, argv []string) {
+	r := rosa.NewRuntime().WithOCM()
+	defer r.Cleanup()
+	err := runWithRuntime(r, cmd, argv)
+	if err != nil {
+		r.Reporter.Errorf(err.Error())
+		os.Exit(1)
+	}
+}
+
+func runWithRuntime(r *rosa.Runtime, cmd *cobra.Command, argv []string) error {
+	clusterKey := r.GetClusterKey()
+	cluster := r.FetchCluster()
+
+	externalAuthService := externalauthprovider.NewExternalAuthService(r.OCMClient)
+	err := externalAuthService.IsExternalAuthProviderSupported(cluster, clusterKey)
+	if err != nil {
+		return err
+	}
+
+	if !interactive.Enabled() {
```

Comment:
@chenz4027 Just checking that we always want to enable interactive mode for this command? If yes, then we can skip the `if !interactive.Enabled()` check and just call `interactive.Enable()`

Reply:
Good catch, nope I should add something like IsExternalAuthProviderSetViaCLI for breakglasscredential. If the parameter is set, we should skip the interactive mode.